config/cluster-api-provider-openshift-assisted/bootstrap/kustomization.yaml

```diff
@@ -12,4 +12,14 @@ patches:
   patch: |-
     - op: replace
       path: /spec/template/spec/containers/0/image
-      value: '{{ .Values.bootstrap.image.url  }}:{{ .Values.bootstrap.image.tag  }}'
\ No newline at end of file
+      value: '{{ .Values.bootstrap.image.url  }}:{{ .Values.bootstrap.image.tag  }}'
+    - op: add
+      path: /spec/template/spec/containers/0/env/-
+      value:
+        name: ASSISTED_CA_BUNDLE_NAME
+        value: default-ingress-cert
```

Comment:
@carbonin could we possibly need to customize this value in runtime, or does it always have the same name? For example, could the user be using some other CA bundle, stored in a different configmap?

Reply:
Not if we're deploying on openshift.